pushapkscript/docker.d/init_worker.sh

```diff
@@ -111,6 +111,7 @@ case $COT_PRODUCT in
         echo $GOOGLE_CREDENTIALS_FOCUS | base64 -d >             $GOOGLE_CREDENTIALS_FOCUS_PATH
         echo $GOOGLE_CREDENTIALS_REFERENCE_BROWSER | base64 -d > $GOOGLE_CREDENTIALS_REFERENCE_BROWSER_PATH
 
+        import_cert fennec-nightly $CERT_DIR/nightly.pem
```

Comment:
Weird that this ceriticate is not present in the `firefox` - `production` section of this file. I wonder why or am I missing something? 🤔 

Later edit: don't mind me, it's imported under a different name at line 66 `import_cert nightly $CERT_DIR/nightly.pem`